libs/wingsdk/src/target-tf-azure/bucket.inflight.ts

```diff
@@ -96,6 +99,14 @@ export class BucketClient implements IBucketClient {
     return list;
   }
 
+  /**
+   * Returns a url to the given file.
+   * @Throws if the file is not public or if object does not exist.
+   */
+  public async publicUrl(key: string): Promise<string> {
+    this._public; // a little help for implementing public_url later on
+    throw new Error(`publicUrl is not supported yet. (key=${key})`);
```

Comment:
Since bucket inflight operations are generally supported on Azure, let's open an issue to track that we do not support `public_url()`.

Reply:
added: https://github.com/winglang/wing/issues/1805